src/libsyntax/feature_gate.rs

```diff
@@ -1310,7 +1310,7 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
                                             stable",
                                             cfg_fn!(profiler_runtime))),
 
-    (sym::allow_internal_unstable, Normal, template!(Word, List: "feat1, feat2, ..."),
+    (sym::allow_internal_unstable, Whitelisted, template!(Word, List: "feat1, feat2, ..."),
```

Comment:
What's this for?

Reply:
gone now, just leftovers from an intermediate state